crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/unspecified_resources.rs

```diff
@@ -0,0 +1,83 @@
+use crate::prelude::*;
+
+/// Represents unspecified resources, which can be either none present or
+/// may be present with a list of change sources.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Enum)]
+pub enum UnspecifiedResources {
+    NonePresent,
+    MayBePresent { change_sources: Vec<ChangeSource> },
```

Comment:
any doc from RET which can be carried over for variants?

Reply:
updated